edweiss-app/app/(app)/courses/[id]/index.tsx

```diff
@@ -370,8 +392,9 @@ const CoursePage: ApplicationRoute = () => {
 				animationType="fade"
 				transparent={true}
 				onRequestClose={() => setActionModalVisible(false)}
+				testID='actionModal'
 			>
-				<Animated.View style={{ flex: 1, justifyContent: 'center', alignItems: 'center', backgroundColor: 'rgba(0, 0, 0, 0.5)', opacity: fadeAnim }}>
+				<Animated.View testID='animatedViewActionSelection' style={{ flex: 1, justifyContent: 'center', alignItems: 'center', backgroundColor: 'rgba(0, 0, 0, 0.5)', opacity: fadeAnim }}>
 					<SelectActions onOutsideClick={closeModalOnOutsideClick} onSelectAssignment={() => handleActionSelect('addAssignment')} onSelectMaterial={() => handleActionSelect('addMaterial')} />
 				</Animated.View>
 			</Modal>
```

Comment:
Using Modal  works, but you could have also used a different type of modal (ModalContainerView), which most of us used already.

Reply:
The Modal that I use, has a different behaviour than ModalContainerView